motor/inventory/v1/inventory.proto

```diff
@@ -129,6 +130,9 @@ message Inventory {
 
   // Most recently observed status of the Inventory.
   InventoryStatus status = 3;
+
+  // optional: the upstream credentials to use for the inventory
+  mondoo.upstream.v1.ServiceAccountCredentials credentials = 4;
```

Comment:
should this be part of the inventory spec?

Reply:
That's probably a better place indeed